tutorials/hyperspectral/Advanced_Spectral_Signatures.Rmd

```diff
@@ -0,0 +1,291 @@
+---
+title: "Advanced Spectral Signatures in R"
```

Comment:
Please use a more detailed title that includes keywords that are connected to the learning objectives. [change file name to match].

Reply:
Yup! Changed this to "Select pixels and compare spectral signatures in R"